src/visualization/types/Map/view.tsx

```diff
@@ -61,52 +71,53 @@ const GeoPlot: FC<Props> = ({result, properties}) => {
     return ''
   }
 
-  switch (errorCode) {
-    case MapErrorStates.MapServiceUnavailable:
-      error =
-        'Map type is having issues connecting to the server. Please try again later.'
-
-      return (
-        <div className="panel-resizer--error" data-testid="geoplot-error">
-          {error}
-        </div>
-      )
-    case MapErrorStates.InvalidCoordinates:
-      error =
-        'Map type is not supported with the data provided: Missing latitude/longitude values'
-
-      return (
-        <div className="panel-resizer--error" data-testid="geoplot-error">
-          {error}
-        </div>
-      )
-    default:
-      const tileServerConfiguration = {
-        tileServerUrl: getMapboxUrl(),
-        bingKey: '',
-      }
+  const coordinatingFieldsFlag = getDetectCoordinatingFields(result.table)
+
+  if (mapServiceError === MapErrorStates.MapServiceUnavailable) {
+    error =
+      'Map type is having issues connecting to the server. Please try again later.'
+
+    return (
+      <div className="panel-resizer--error" data-testid="geoplot-error">
+        {error}
+      </div>
+    )
+  } else if (coordinateError === MapErrorStates.InvalidCoordinates) {
+    error =
+      'Map type is not supported with the data provided: Missing latitude/longitude values (field values must be specified to lat or lon)'
+
+    return (
+      <div className="panel-resizer--error" data-testid="geoplot-error">
+        {error}
+      </div>
+    )
+  } else {
```

Comment:
✂️ the `else` - this is the main 'happy path' of the code. it should be the first level of indentation, not squirreled away in an `else` statement. those `return` statements in the `if` statements make `else` (and `else if`) unnecessary